router/internal/circuit/manager.go

```diff
@@ -0,0 +1,108 @@
+package circuit
+
+import (
+	"fmt"
+	"github.com/cep21/circuit/v4"
+	"github.com/cep21/circuit/v4/closers/hystrix"
+	nodev1 "github.com/wundergraph/cosmo/router/gen/proto/wg/cosmo/node/v1"
+	"github.com/wundergraph/cosmo/router/pkg/config"
```

Comment:
This pkg should not rely on router config structs. Keep it decoupled.

Reply:
This is done.